crates/glaredb/src/args/server.rs

```diff
@@ -108,4 +101,17 @@ pub struct ServerArgs {
     /// This will fully disable the postgres server on port 6543.
     #[arg(long, default_value="false", action = clap::ArgAction::SetTrue)]
     pub disable_postgres_api: bool,
+
+    /// Bucket to use for database catalogs.
+    #[clap(long, value_parser)]
+    pub metastore_bucket: Option<String>,
+
+    /// Path to GCP service account to use when connecting to GCS.
+    #[clap(long, value_parser)]
+    pub metastore_service_account_path: Option<String>,
```

Comment:
These are the two main args that power it on cloud 👍🏻

Reply:
I think we need to delete the `service_account_path` because it should be pulled from the environment credentials, we only really need the bucket name, I would think. 

File path is only for "local" operation, and we can/should leave that.

Reply:
Yep, sounds like a plan

Reply:
I took out the metastore service account so we're just to bucket name (or filepath, if we want that to be configured, though in that case we could just put `__metastore` in the data directory...